server/src/main/java/org/gluu/oxtrust/api/CustomScriptManagementService.java

```diff
@@ -0,0 +1,400 @@
+/*
+ * oxTrust is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2014, Gluu
+ */
+
+package org.gluu.oxtrust.api;
+
+import com.google.common.base.Optional;
+import com.wordnik.swagger.annotations.ApiOperation;
+import com.wordnik.swagger.annotations.ApiResponse;
+import com.wordnik.swagger.annotations.ApiResponses;
+import org.gluu.oxtrust.exception.InvalidScriptDataException;
+import org.gluu.oxtrust.exception.ScriptNotFoundException;
+import org.gluu.oxtrust.ldap.service.ApplianceService;
+import org.gluu.oxtrust.ldap.service.OrganizationService;
+import org.gluu.oxtrust.model.CustomScriptDTO;
+import org.gluu.oxtrust.service.filter.ProtectedApi;
+import org.gluu.oxtrust.util.OxTrustApiConstants;
+import org.gluu.persist.exception.BasePersistenceException;
+import org.slf4j.Logger;
+import org.xdi.model.ScriptLocationType;
+import org.xdi.model.custom.script.CustomScriptType;
+import org.xdi.model.custom.script.model.CustomScript;
+import org.xdi.model.custom.script.model.ScriptError;
+import org.xdi.service.custom.script.AbstractCustomScriptService;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.singletonList;
+import static org.apache.commons.lang.StringUtils.isNotEmpty;
+import static org.gluu.oxtrust.action.ManageCustomScriptAction.*;
+import static org.gluu.oxtrust.model.CustomScriptDTO.toDTO;
+
+/**
+ * Service for custom script management
+ *
+ * @author Shoeb Khan
+ * @version October 24, 2018
+ */
+
+
+@Path(OxTrustApiConstants.BASE_API_URL + "/configurations/scripts")
+@ProtectedApi
```

Comment:
add @Produces(MediaType.APPLICATION_JSON), @Consumes(MediaType.APPLICATION_JSON) and remove the @Produces from the methods